tests/CMakeLists.txt

```diff
@@ -52,4 +52,8 @@ target_link_libraries( es_test
                        graphene_es_objects graphene_egenesis_none graphene_api_helper_indexes
                        fc ${PLATFORM_SPECIFIC_LIBS} )
 
+file( GLOB VOTING_STAT_SOURCES "voting_stat/*.cpp" )
+add_executable( voting_stat_test ${COMMON_SOURCES} ${VOTING_STAT_SOURCES} )
```

Comment:
Please move general tests into normal test suite and ES-specific tests into ES test suite. A separate executable will most likely not be run by developers.

Reply:
Okay will do that.